CondTools/CTPPS/plugins/CTPPSPixGainCalibsESAnalyzer.cc

```diff
@@ -15,8 +15,8 @@
 class CTPPSPixGainCalibsESAnalyzer : public edm::one::EDAnalyzer<> {
 public:
   explicit CTPPSPixGainCalibsESAnalyzer(edm::ParameterSet const& p)
-      : m_outfilename(p.getUntrackedParameter<std::string>("outputrootfile", "output.root")) {
-    //    std::cout<<"CTPPSPixGainCalibsESAnalyzer"<<std::endl;
+      : m_outfilename(p.getUntrackedParameter<std::string>("outputrootfile", "output.root")),
+        tokenCalibration_(esConsumes<CTPPSPixelGainCalibrations, CTPPSPixelGainCalibrationsRcd>()) {
```

Comment:
For future reference, this could have been `tokenCalibration_(esConsumes())`. The code is 'smart enough' to determine what the type is based on the type of the token.